frontend/src/repo-menu.tsx

```diff
@@ -49,11 +53,18 @@ export function RepoMenu() {
                 key={repo}
                 value={repo}
                 isChecked={showAll || selectedReposSet.has(repo)}>
-                {repo}
+                {repo} ({repoToPullCount.get(repo) || 0})
              </MenuItemOption>
           )}
        </MenuOptionGroup>
      </MenuList>
    </Menu>
    );
 }
+
+function getRepoToPullCount(pulls: Pull[]): RepoCounts {
+   return pulls.reduce((counts, pull) => {
+      counts.set(pull.getRepoName(), (counts.get(pull.getRepoName()) || 0) + 1);
+      return counts;
+   }, new Map() as RepoCounts);
```

Comment:
Might be worth doing this as a [`countBy`](https://lodash.com/docs/4.17.15#countBy), since it makes it clear exactly what we're doing:
```
return _.countBy(pulls, pull => pull.getRepoName());
```
(you'll need to switch the return type to `Record<string, number>` or something like that, I think).

Reply:
Good idea, fixed!